README.md

```diff
@@ -1,42 +1,63 @@
 
 TypeDB Studio is TypeDB's Integrated Development Environment to perform knowledge engineering.
 
-You can utilise Studio like [TypeDB Console](http://docs.vaticle.com/docs/running-typedb/console)
-and [TypeDB Clients](http://docs.vaticle.com/docs/client-api/overview), to connect to your TypeDB instance and perform
+You can utilise Studio like [TypeDB Console](https://docs.vaticle.com/docs/console/console)
+and [TypeDB Clients](https://docs.vaticle.com/docs/client-api/overview), to connect to your TypeDB instance and perform
 queries.
 
-Whether you need a tool to test and experiment with your newly created TypeDB database, or you prefer a graphical
-interface for reading data from TypeDB, you will find Studio extremely useful.
+Studio is an IDE designed for the end-to-end development of TypeDB databases, database queries, and data exploration, 
+via a clean and intuitive UI.
 
-## Alpha Releases
+## Download & Install
+TypeDB Studio is available for Linux, Mac and Windows. Head over to the 
+[releases](https://github.com/vaticle/typedb-studio/releases) page to download and install the latest release of Studio.
 
-At this time, alpha releases allow you to visualise and explore your data!
+## Usage
+Upon launching Studio, you'll be asked for your TypeDB server details and to choose or create a database. Studio makes
+these choices explicit so that it's always clear what you're working on at any given time.
 
-Styling, layouts, installation, etc. are all subject to change.
+### Manage TypeDB Projects
 
-## Usage
+After connecting to TypeDB and selecting a database, Studio will prompt you to open (or create) a project directory. 
+This is where Studio will load and save query files, but we can also freely edit text files in the project directory 
+using Studio. The project browser maintains a live view of the directory contents on the filesystem, much like other 
+major IDEs do.
 
-Usage is easy! After you install the appropriate distribution, just enter the address and name of your database and
-start writing queries.
+### Manage Database Schemas
+[![Log Output](./images/type_browser_1.png)](./images/type_browser_1.png)
 
-### Reasoning and Explanations
+The Type Browser provides a structured tree view of the connected database's type hierarchy. Opening any Type will 
+bring up a page that lists in detail its properties - supertype, roles played, attributes owned, etc.
 
-TypeDB Studio currently always enables inference, and highlights the concepts which are inferred when visualising
-answers to your query.
+### Run TypeQL Queries
+[![Log Output](./images/log_output_1.png)](./images/log_output_1.png)
 
-Double click the inferred concepts to retrieve their explanations and visualise how the fact was inferred.
+Studio sports a fully-featured, performant text editor including all the functionality one would expect from an editor 
+(Undo/Redo, Find/Replace, etc.) It also includes syntax highlighting for TypeQL files (\*.tql). Studio's intuitive and 
```

Comment:
This is a valuable section to have for sure, but I'd say let's tune it down to be less salesy:
```suggestion
Studio's text editor comes equipped with rich developer tooling, like syntax highlighting
for TypeQL files (\*.tql), with more advanced features currently under development. Studio's intuitive and 
```